bash.md

```diff
@@ -57,6 +57,20 @@ Remove the variable:
 unset my_variable
 ```
 
+Define Variables using export
+
+```
+export my_variable='Hello from Bash!'
+```
+
+Remove variable defined by export
+
+```
+export -n my_variable
```

Comment:
Can you add any doc reference to this? Want to cross-check the correctness

Reply:
https://www.baeldung.com/linux/bash-variables-export